MAINTAINERS.md

```diff
@@ -0,0 +1,37 @@
+# Podman Maintainers
+
+[GOVERNANCE.md](https://github.com/containers/podman/blob/main/GOVERNANCE.md)
+describes the Podman project's governance and the Project Roles used below.
+
+## Maintainers
+
+| Maintainer           | GitHub ID                                                | Project Roles     | Affiliation                               |
+| -------------------- | -------------------------------------------------------- | ----------------- | ----------------------------------------- |
+
+| Brent Baude          | [baude](https://github.com/baude)                        | Core Maintainer   | [Red Hat](https://www.github.com/redhat/) |
+| Ygal Blum            | [ygalblum](https://github.com/ygalblum)                  | Maintainer        | [Red Hat](https://www.github.com/redhat/) |
+| Jake Correnti        | [jakecorrenti](https://github.com/jakecorrenti)          | Reviewer          | [Red Hat](https://www.github.com/redhat/) |
+| Ashley Cui           | [ashley-cui](https://github.com/ashley-cui)              | Maintainer        | [Red Hat](https://www.github.com/redhat/) |
+| Nalin Dahyabhai      | [nalind](https://github.com/nalind)                      | Core Maintainer   | [Red Hat](https://www.github.com/redhat/) |
+| Jason Greene         | [n1hility](https://github.com/n1hility)                  | Reviewer          | [Red Hat](https://www.github.com/redhat/) |
+| Matthew Heon         | [mheon](https://github.com/mheon)                        | Core Maintainer   | [Red Hat](https://www.github.com/redhat/) |
+| Paul Holzinger       | [luap99](https://github.com/Luap99)                      | Core Maintainer   | [Red Hat](https://www.github.com/redhat/) |
+| Jhon Honce           | [jwhonce](https://github.com/jwhonce)                    | Reviewer          | [Red Hat](https://www.github.com/redhat/) |
+| Mario Loriedo        | [l0rd](https://github.com/l0rd/)                         | Maintainer        | [Red Hat](https://www.github.com/redhat/) |
+| Lokesh Mandvekar     | [lsm5](https://github.com/lsm5)                          | Maintainer        | [Red Hat](https://www.github.com/redhat/) |
+| Urvashi Mohnani      | [umohnani8](https://github.com/umohnani8)                | Reviewer          | [Red Hat](https://www.github.com/redhat/) |
+| Aditya Rajan         | [flouthoc](https://github.com/flouthoc)                  | Reviewer          | [Red Hat](https://www.github.com/redhat/) |
+| Valentin Rothberg    | [vrothberg](https://github.com/vrothberg)                | Reviewer          | [Red Hat](https://www.github.com/redhat/) |
+| Giuseppe Scrivano    | [giuseppe](https://github.com/giuseppe)                  | Core Maintainer   | [Red Hat](https://www.github.com/redhat/) |
+| Neil Smith           | [Neil-Smith](https://github.com/Neil-Smith)              | Community Manager | [Red Hat](https://www.github.com/redhat/) |
```

Comment:
Being "Community Manager" but per the governance a "Community Manager" is also a Maintainer and I think that is not right for Neil as he doesn't fulfil other important Maintainer tasks.

And another thing that is unclear if one is listed as "Community Manager" are they a normal maintainer or core maintainer? Maybe it would be better to define  "Community Manager" as a role that is not tied to the maintainer/reviewer roles.
Then someone can only be a Community Manager. And for a Maintainer such as Tom we could just list as Community Manager and Core Maintainer?

Reply:
I originally had Community Manager as a separate role, but moved it into Maintainer at the suggestion of @baude - I have no issues with separating it again.

Reply:
I think that is mischaracterized a bit.  My opinion is that a community manager should not just be anyone off the street.  As such, I would agree that community manager in itself does not entitle you to privileges to the repository.  In other words, you can be a community manager and maintainer; but being a community manager does not make you maintainer.

Reply:
Updated to make Community Manager a separate role.